cli/reference/coder.md

```diff
@@ -0,0 +1,34 @@
+<!-- markdownlint-disable MD044 -->
+
+# coder
+
+coder provides a CLI for working with an existing Coder installation
+
+### Options
+
+```text
+  -h, --help      help for coder
+  -v, --verbose   show verbose output
+```
+
+### SEE ALSO
+
+- [coder completion](coder_completion.md) - Generate completion script
```

Comment:
These 404 for me in preview; I wonder if they also need to be relative links? e.g.

```suggestion
- [coder completion](./coder_completion.md) - Generate completion script
```

Reply:
Yeah, I think it's the manifest.  Just pushed a fix